pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -8603,6 +8617,13 @@
   hoc_how_to_step_companies_choose_activities_desc_03: "Then, [learn all about AI](%{ai_url}). Employees can watch short videos featuring experts in the field."
   hoc_how_to_step_companies_spread_word_desc_01: "After your event, congratulate your employees with personalized certificates of completion. Share photos and videos of your Hour of Code event and show off your commitment to supporting the computer science movement using #HourOfCode and @codeorg."
 
+  hoc_how_to_step_districts_recruit_desc_01: "Recruit school participation by sharing the resources provided in our how-to guides for educators and school assemblies. With your support, teachers and school administrators can decide which type of event best meets the interest of your schools."
+  hoc_how_to_step_districts_register_desc_01: "Tell everyone hosting an event to register! When you register your Hour of Code event, you'll receive helpful emails with news and tips for hosting a successful Hour of Code. It's also how you can let local volunteers know your school is participating. Volunteers are a great resource and can come speak to your class about computer science or help your students with Hour of Code activities."
+  hoc_how_to_step_districts_promote_desc_01: "Let community members and stakeholders know about your participation in Hour of Code Events! Post to Facebook, Twitter, Instagram and your district's homepage! Explore our sample emails which help you share your Hour of Code event with your community."
+  hoc_how_to_step_districts_check_out_ai_desc_01: "Demystify artificial intelligence (AI) by learning how it's changing the ways we learn and engage our students. \"How AI Works\" provides opportunities for districts to deepen their understanding of AI through professional learning opportunities such as, \"A1 101 for Teachers\" and through [lessons on How AI Works](%{lessons_url}). These tools will help districts, teachers and students learn how to more effectively understand and utilize AI."
```

Comment:
I think this says "A1 101" instead of "AI 101"?

Reply:
Fixed!